plugins/inputs/http_listener/http_listener.go

```diff
@@ -75,6 +83,18 @@ const sampleConfig = `
   ## Maximum line size allowed to be sent in bytes.
   ## 0 means to use the default of 65536 bytes (64 kibibytes)
   max_line_size = 0
+
+  ## Set one or more allowed client CA certificate file names to 
+  ## enable mutually authenticated TLS connections
+  ssl_allowed_client_certificate_authorities = ["/etc/ca.crt"]
+
+  ## Add non-public root of trust certificate authorities
+  ssl_certificate_authorities = ["/etc/ca.crt"]
```

Comment:
This is probably a limitation in my TLS understanding, but I thought this (the one assigned to `RootCAs`) was only needed if you were a client.  For instance, when setting a basic nginx server without client authentication up, you only need the `ssl_certificate` and `ssl_certificate_key`.  Why do we need both of these CA options?